### ISSUE TITLE
[Spell] Blink

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#3532]
ID 1953

New issue that arose recently, the spell used to work more or less fine up til a week ago.

**How it should work:** Blink should make you travel up to 20 yards forward until your way is blocked by an obstacle. If an obstacle or an edge of a cliff is within 20 yrds, it should still let you travel as far as possible, stopping you where it can no longer move you past. 
The only case when it should downright fail to move you at all is if you're standing on the edge of a cliff, because the spell was designed to be unable to make you fly through air.
**How it works**: Blink seems to now fail in various terrain spots for no reason, making the spell useless and sabotage the PVP life of mages. 
I made some screens from random spots in dalaran where casting blink results in your character staying in place:
![WoWScrnShot_082619_222131](https://user-images.githubusercontent.com/31132743/63721421-89b8b300-c851-11e9-821d-47648f4389aa.jpg)
![WoWScrnShot_082619_222116](https://user-images.githubusercontent.com/31132743/63721436-91785780-c851-11e9-9f09-9ebe1b41d269.jpg)
![WoWScrnShot_082619_222226](https://user-images.githubusercontent.com/31132743/63721460-9c32ec80-c851-11e9-917a-d6cabcf52ae2.jpg)

There is zero reason for your spell to fail in any of these spots, there isn't a single pebble or else in your way. And this is happening in BGs, arenas, in many crucial spots where mages need to gain distance.
Please revert the spell to the previous state if you cannot fix it, I am sure something was changed in the recent days because it used to work fine.
